server/router/server.go

```diff
@@ -345,7 +345,8 @@ func NewRouter(_ context.Context, h models.HandlerInterface, port int, g http.Ha
 
 	gMux.Handle("/api/identity/orgs", h.ProviderMiddleware(h.AuthMiddleware(h.SessionInjectorMiddleware(h.GetOrganizations), models.ProviderAuth))).
 		Methods("GET")
-
+	gMux.Handle("/api/meshmodel/download", h.ProviderMiddleware(h.AuthMiddleware(http.HandlerFunc(h.DownloadEntity), models.NoAuth))).Methods("GET")
```

Comment:
Please name the APIs meaningfully, are you downloading the entire models (no, correct).